Git-Collabration-Example/ViewController.swift

```diff
@@ -9,11 +9,30 @@ import UIKit
 
 class ViewController: UIViewController {
 
+    @IBOutlet weak var clickDisplay: UIButton!
+    @IBOutlet weak var commentService: UILabel!
     override func viewDidLoad() {
         super.viewDidLoad()
         // Do any additional setup after loading the view.
+        
+        
     }
-
+    
+    
+    @IBAction func showPage(_ sender: UIButton, forEvent event: UIEvent) {
+        let currentLabel = commentService.text
+        if (currentLabel == "You just click a button. ") {
+            commentService.text = "You click the button again"
+        }
+        else
+        {
+            commentService.text = "You just click a button"
+        }
+        
+        commentService.sizeToFit()
+    }
+    
+    
```

Comment:
Please remove excess space on lines 34 and 35 as a practice on resolving reviews

Reply:
It'll be taken care shortly

Reply:
Done

Reply:
I still see whitelines on this. Is it just me?

Reply:
It's done now and committed, unless I'm missing something as well